packages/core/src/common/cluster-types.ts

```diff
@@ -105,6 +105,7 @@ export interface ClusterPreferences extends ClusterPrometheusPreferences {
    * cleared by preferences.
    */
   icon?: string | null;
+  iconBackgroundColor?: string;
```

Comment:
Why do we need another field for `icon`? It's possible I guess to save some `string` with color (HEX/rgba) to `icon` field too and distinguish them easily (when it's not empty) by `prefs.icon.startsWith("data:")` to proper show background.